### PR TITLE
LPS-98188 Apply the new method

### PR DESCRIPTION
**Pull request by leoadb**

### CODE REVIEW

**Comment by leoadb** [comment]
ci:test:sf

**Comment by leoadb** [comment]
ci:test:relevant

**Comment by leoadb** [comment]
ci:test:sf

**Comment by leoadb** [comment]
ci:test:relevant

**Comment by leoadb** [comment]
ci:test:sf

File: modules/apps/data-engine/data-engine-rest-impl/package.json [hunk 1]
```diff
@@ -0,0 +1,29 @@
+{
+	"dependencies": {
```
Comment (brunofarache) [inline modules/apps/data-engine/data-engine-rest-impl/package.json]: @leoadb check if dependencies are needed

File: modules/apps/data-engine/data-engine-rest-impl/package.json [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+		"metal-state": "2.16.8",
+		"object-hash": "^1.3.0"
+	},
+	"name": "data-engine-rest-impl",
+	"private": true,
+	"scripts": {
+		"build": "liferay-npm-scripts build",
+		"checkFormat": "liferay-npm-scripts check",
+		"format": "liferay-npm-scripts fix",
+		"test": "liferay-npm-scripts test"
```
Comment (brunofarache) [inline modules/apps/data-engine/data-engine-rest-impl/package.json]: @leoadb copy from app-builder-web. remove test and checkFormat scripts.

File: modules/apps/data-engine/data-engine-rest-impl/package.json [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+	},
+	"version": "2.0.0"
```
Comment (brunofarache) [inline modules/apps/data-engine/data-engine-rest-impl/package.json]: @leoadb should be 1.0.0?

File: modules/apps/data-engine/data-engine-rest-impl/src/main/java/com/liferay/data/engine/rest/internal/renderer/v1_0/DataLayoutRendererImpl.java [hunk 4]
```diff
@@ -268,6 +443,11 @@ public String render(
 	@Reference
 	private NPMResolver _npmResolver;
 
+	@Reference
+	private Portal _portal;
+
+	private Map<Locale, ResourceBundle> _resourceBundles;
```
Comment (brunofarache) [inline modules/apps/data-engine/data-engine-rest-impl/src/main/java/com/liferay/data/engine/rest/internal/renderer/v1_0/DataLayoutRendererImpl.java]: @leoadb final new HashMap()

**Pull request closed without merging**